src/lib/parse_keys.js

```diff
@@ -1,14 +1,16 @@
-import Keys, { modifiers } from './keys';
+import Keys, { modifiers, ALL_KEYS } from './keys';
 
 function parseKeys( keysArray ) {
+  if (ALL_KEYS === keysArray[0]) return
```

Comment:
it'd be nice not to have special cases in here if we can avoid it, and we'd like a more consistent return value.